src/main/java/net/masterthought/cucumber/generators/AbstractPage.java

```diff
@@ -79,6 +80,7 @@ private Properties buildProperties() {
         props.setProperty("resource.loader", "class");
         props.setProperty("class.resource.loader.class", ClasspathResourceLoader.class.getCanonicalName());
         props.setProperty("runtime.log", new File(configuration.getReportDirectory(), "velocity.log").getPath());
+        props.setProperty("eventhandler.referenceinsertion.class", "org.apache.velocity.app.event.implement.EscapeHtmlReference");
```

Comment:
I'm sorry. I can't quite parse your intentions.

Should the report look like this

> 
> Scenario: A scenario
> 
> <i>Some text here with safe html markup.</i>
> 
> Feature: Docstring
>    Given a doc string containing html mark up
>    """
>   \<i\>Some text here with html markup\</i\>
>    """
> 

Or like this:

> 
> Scenario: A scenario
> 
> \<i\>Some text here with safe html markup.\</i\>
> 
> Feature: Docstring
>    Given a doc string containing html mark up
>    """
>   \<i\>Some text here with html markup\</i\>
>    """
>